### PR TITLE
Add allUsers to cloud run invokers to allow public access to metrics …

### PR DESCRIPTION
**Pull request by pdewilde**
…server

### CODE REVIEW

File: terraform/cloudrun.tf [hunk 1]
```diff
@@ -57,6 +57,17 @@ resource "google_cloud_run_v2_service" "metrics" {
   }
 }
 
+resource "google_cloud_run_v2_service_iam_binding" "public_metrics_access" {
```
Comment (gjonathanhong) [inline terraform/cloudrun.tf:60]: Please use the `_iam_member` resource instead of `_iam_binding`

Comment (pdewilde) [reply terraform/cloudrun.tf]: Done

File: terraform/cloudrun.tf [hunk 2]
```diff
@@ -57,6 +57,17 @@ resource "google_cloud_run_v2_service" "metrics" {
   }
 }
 
+resource "google_cloud_run_v2_service_iam_member" "public_metrics_access" {
+  project = var.project_id
+
+  location = google_cloud_run_v2_service.metrics.location
+  name     = google_cloud_run_v2_service.metrics.name
+  role     = "roles/run.invoker"
+  members = [
```
Comment (gjonathanhong) [inline terraform/cloudrun.tf:66]: Unlike `_iam_binding`, `_iam_member` does not allow a list of members as an argument. You will need to use a `for_each` instead, or "allUsers" directly.

Comment (pdewilde) [reply terraform/cloudrun.tf]: Was too fast at my documentation reading it seems, thanks

**Review by gjonathanhong — APPROVED** [review]

**Pull request merged**